packages/components/inputs/select-input/src/select-input.tsx

```diff
@@ -405,7 +417,9 @@ const SelectInput = (props: TSelectInputProps) => {
   });
 
   const placeholder =
-    props.placeholder || intl.formatMessage(messages.placeholder);
+    props.appearance === 'filter'
+      ? intl.formatMessage(messages.selectInputAsFilterPlaceholder)
+      : props.placeholder || intl.formatMessage(messages.placeholder);
```

Comment:
Is this correct? When `appearance` is set to `filter`, the `placeholder` prop is essentially ignored.

Reply:
Thats a good question 🤔, from the design, it may seem like we want that to be search, but on a second thought now, I think I will change the logic to still capture a provided placeholder from prop. Thank you.

Reply:
I made the placeholder prop visible when provided, otherwise, use the default messages provided here: https://github.com/commercetools/ui-kit/pull/2975/commits/86acbb581fc8050262485455bcf4dcd45a5a5d68